segway/output.py

```diff
@@ -54,7 +54,10 @@ def concatenate_window_segmentations(window_filenames, header, outfilename):
                 # write the last line and the first line, after
                 # potentially merging
                 if last_vals == (chrom, start, seg):
+                    # update start position
                     first_row[INDEX_BED_START] = last_start
+                    # update thickStart position
+                    first_row[INDEX_BED_START + 5] = last_start
```

Comment:
change indexing to a constant to something like `INDEX_BED_THICKSTART` and have it defined in terms of `INDEX_BED_START`